tarteaucitron.services.js

```diff
@@ -1848,3 +1848,48 @@ tarteaucitron.services.facebookpixel = {
     }
 };
 
+//Issuu
+tarteaucitron.services.issuu = {
+    "key": "issuu",
+    "type": "other",
```

Comment:
I don't know this service. That's better to avoid "other". Ads is ok?

Reply:
Issuu is a service to display PDFs embeded into your website. 
So, "ads" is not the right place to put it, I think.

Reply:
Ok, set the type to video like calameo and prezi and I will change this later to "media".